src/main/java/com/gitblit/Constants.java

```diff
@@ -411,7 +411,7 @@ public String toString() {
 		LIST_REPOSITORY_MEMBERS, SET_REPOSITORY_MEMBERS, LIST_REPOSITORY_TEAMS, SET_REPOSITORY_TEAMS,
 		LIST_REPOSITORY_MEMBER_PERMISSIONS, SET_REPOSITORY_MEMBER_PERMISSIONS, LIST_REPOSITORY_TEAM_PERMISSIONS, SET_REPOSITORY_TEAM_PERMISSIONS,
 		LIST_FEDERATION_REGISTRATIONS, LIST_FEDERATION_RESULTS, LIST_FEDERATION_PROPOSALS, LIST_FEDERATION_SETS,
-		EDIT_SETTINGS, LIST_STATUS;
+		EDIT_SETTINGS, LIST_STATUS, RUN_GC_WHERE_NEEDED, RUN_SINGLE_GC;

```

Comment:
Unify `RUN_GC_WHERE_NEEDED` and `RUN_SINGLE_GC` to `COLLECT_GARBAGE' and do logic like for `REINDEX_TICKETS`.